Homebrew installation broken

When installing accio as per the readme,
i get the following error, can anyone reproduce?

MacOS: 10.15.6
Xcode: 12 beta 6

```shell
 brew install accio
==> Installing accio from jamitlabs/accio
==> Cloning https://github.com/JamitLabs/Accio.git
Updating /Users/fawi/Library/Caches/Homebrew/accio--git
==> Checking out tag 0.6.5
HEAD is now at 39cc8c6 Merge branch 'deploy/0.6.5' into productive
HEAD is now at 39cc8c6 Merge branch 'deploy/0.6.5' into productive
==> make install prefix=/usr/local/Cellar/accio/0.6.5
Last 15 lines from /Users/fawi/Library/Logs/Homebrew/accio/01.make:
15 swift                    0x00000001105e6672 swift::WitnessChecker::findBestWitness(swift::ValueDecl*, bool*, swift::NormalProtocolConformance*, llvm::SmallVectorImpl<swift::RequirementMatch>&, unsigned int&, unsigned int&, bool&) + 946
16 swift                    0x00000001105f0ce3 swift::ConformanceChecker::resolveWitnessViaLookup(swift::ValueDecl*) + 643
17 swift                    0x00000001105ebbc5 swift::MultiConformanceChecker::checkIndividualConformance(swift::NormalProtocolConformance*, bool) + 14773
18 swift                    0x00000001105e7cb3 swift::MultiConformanceChecker::checkAllConformances() + 147
19 swift                    0x00000001105f7837 swift::TypeChecker::checkConformancesInContext(swift::DeclContext*, swift::IterableDeclContext*) + 8295
20 swift                    0x00000001105a7c4a (anonymous namespace)::DeclChecker::visitClassDecl(swift::ClassDecl*) + 8138
21 swift                    0x00000001105a0143 (anonymous namespace)::DeclChecker::visit(swift::Decl*) + 2739
22 swift                    0x0000000110667811 swift::TypeCheckSourceFileRequest::evaluate(swift::Evaluator&, swift::SourceFile*) const + 753
23 swift                    0x000000011066a699 llvm::Expected<swift::TypeCheckSourceFileRequest::OutputType> swift::Evaluator::getResultUncached<swift::TypeCheckSourceFileRequest>(swift::TypeCheckSourceFileRequest const&) + 953
24 swift                    0x0000000110667394 swift::TypeCheckSourceFileRequest::OutputType swift::evaluateOrDefault<swift::TypeCheckSourceFileRequest>(swift::Evaluator&, swift::TypeCheckSourceFileRequest, swift::TypeCheckSourceFileRequest::OutputType) + 164
25 swift                    0x000000010f7e2e78 swift::CompilerInstance::performSemaUpTo(swift::SourceFile::ASTStage_t) + 7576
26 swift                    0x000000010f6a396d swift::performFrontend(llvm::ArrayRef<char const*>, char const*, void*, swift::FrontendObserver*) + 6845
27 swift                    0x000000010f628277 main + 1255
28 libdyld.dylib            0x00007fff69a60cc9 start + 1
make: *** [accio] Error 1

If reporting this issue please do so to (not Homebrew/brew or Homebrew/core):
  jamitlabs/accio

Traceback (most recent call last):
	22: from /usr/local/Homebrew/Library/Homebrew/build.rb:229:in `<main>'
	21: from /usr/local/Homebrew/Library/Homebrew/build.rb:140:in `install'
	20: from /usr/local/Homebrew/Library/Homebrew/utils.rb:492:in `with_env'
	19: from /usr/local/Homebrew/Library/Homebrew/build.rb:145:in `block in install'
	18: from /usr/local/Homebrew/Library/Homebrew/formula.rb:1175:in `brew'
	17: from /usr/local/Homebrew/Library/Homebrew/formula.rb:2149:in `stage'
	16: from /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/2.6.0/forwardable.rb:230:in `stage'
	15: from /usr/local/Homebrew/Library/Homebrew/resource.rb:86:in `stage'
	14: from /usr/local/Homebrew/Library/Homebrew/resource.rb:111:in `unpack'
	13: from /usr/local/Homebrew/Library/Homebrew/resource.rb:201:in `mktemp'
	12: from /usr/local/Homebrew/Library/Homebrew/mktemp.rb:57:in `run'
	11: from /usr/local/Homebrew/Library/Homebrew/mktemp.rb:57:in `chdir'
	10: from /usr/local/Homebrew/Library/Homebrew/mktemp.rb:57:in `block in run'
	 9: from /usr/local/Homebrew/Library/Homebrew/resource.rb:116:in `block in unpack'
	 8: from /usr/local/Homebrew/Library/Homebrew/formula.rb:2169:in `block in stage'
	 7: from /usr/local/Homebrew/Library/Homebrew/utils.rb:492:in `with_env'
	 6: from /usr/local/Homebrew/Library/Homebrew/formula.rb:2170:in `block (2 levels) in stage'
	 5: from /usr/local/Homebrew/Library/Homebrew/formula.rb:1182:in `block in brew'
	 4: from /usr/local/Homebrew/Library/Homebrew/build.rb:177:in `block (2 levels) in install'
	 3: from /usr/local/Homebrew/Library/Taps/jamitlabs/homebrew-accio/Formula/accio.rb:10:in `install'
	 2: from /usr/local/Homebrew/Library/Homebrew/formula.rb:1963:in `system'
	 1: from /usr/local/Homebrew/Library/Homebrew/formula.rb:1963:in `open'
/usr/local/Homebrew/Library/Homebrew/formula.rb:2025:in `block in system': Failed executing: make install prefix=/usr/local/Cellar/accio/0.6.5 (BuildError)
	9: from /usr/local/Homebrew/Library/Homebrew/brew.rb:148:in `<main>'
	8: from /usr/local/Homebrew/Library/Homebrew/brew.rb:160:in `rescue in <main>'
	7: from /usr/local/Homebrew/Library/Homebrew/exceptions.rb:439:in `dump'
	6: from /usr/local/Homebrew/Library/Homebrew/exceptions.rb:385:in `issues'
	5: from /usr/local/Homebrew/Library/Homebrew/exceptions.rb:389:in `fetch_issues'
	4: from /usr/local/Homebrew/Library/Homebrew/utils/github.rb:316:in `issues_for_formula'
	3: from /usr/local/Homebrew/Library/Homebrew/utils/github.rb:304:in `search_issues'
	2: from /usr/local/Homebrew/Library/Homebrew/utils/github.rb:411:in `search'
	1: from /usr/local/Homebrew/Library/Homebrew/utils/github.rb:229:in `open_api'
/usr/local/Homebrew/Library/Homebrew/utils/github.rb:284:in `raise_api_error': Validation Failed: [{"message"=>"The listed users and repositories cannot be searched either because the resources do not exist or you do not have permission to view them.", "resource"=>"Search", "field"=>"q", "code"=>"invalid"}] (GitHub::ValidationFailedError)
```

--- a/Package.resolved
+++ b/Package.resolved
@@ -78,8 +78,8 @@
         "repositoryURL": "https://github.com/kareman/SwiftShell.git",
         "state": {
           "branch": null,
-          "revision": "beebe43c986d89ea5359ac3adcb42dac94e5e08a",
-          "version": "4.1.2"
+          "revision": "99680b2efc7c7dbcace1da0b3979d266f02e213c",
+          "version": "5.1.0"
         }
       },
       {

--- a/Package.swift
+++ b/Package.swift
@@ -14,7 +14,7 @@ let package = Package(
         .package(url: "https://github.com/Flinesoft/HandySwift.git", .upToNextMajor(from: "3.0.0")),
         .package(url: "https://github.com/onevcat/Rainbow.git", .upToNextMajor(from: "3.1.4")),
         .package(url: "https://github.com/jakeheis/SwiftCLI.git", .upToNextMajor(from: "5.2.2")),
-        .package(url: "https://github.com/kareman/SwiftShell.git", .upToNextMajor(from: "4.1.2")),
+        .package(url: "https://github.com/kareman/SwiftShell.git", .upToNextMajor(from: "5.1.0")),
         .package(url: "https://github.com/tuist/xcodeproj.git", .upToNextMajor(from: "7.0.0")),
     ],
     targets: [

